cmd/log-parser/README.md

```diff
@@ -0,0 +1,69 @@
+# `kata-log-parser`
+
+* [Component logfiles](#component-logfiles)
+* [Usage](#usage)
+
+`kata-log-parser` is a tool that combines log files from various Kata
+Containers components. It sorts the logfiles by timestamp and re-displays
+them, adding a time delta showing how much time has elapsed between each log
+entry.
+
+It is checks the logs for validity and can re-format the logs and output them in a different format.
+
+For full details,
+
+```
+$ kata-log-parser --help
+```
+
+## Component logfiles
+
+The primary logfiles the tool reads are:
```

Comment:
I presume the tool itself is component neutral? It will work with any set of appropriately formatted log files I think?

Reply:
I've added a new section to the doc explaining the tools expectations.